lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/BaseLuceneLock.java

```diff
@@ -34,10 +36,6 @@
       this.keyOfLock = new FileCacheKey(indexName, lockName);
    }
 
-   /**
-    * {@inheritDoc}
-    */
-   @Override
```

Comment:
The method was removed from the Lucene API (it's no longer on the extended `org.apache.lucene.store.Lock`) and I initially didn't have the `org.infinispan.lucene.impl.ObtainableLock`.

Since I ended up introducing `org.infinispan.lucene.impl.ObtainableLock` you could restore the Override annotation if you prefer.